hopkin/routes/customer.py

```diff
@@ -51,7 +51,6 @@ def customer_profile_info() -> dict:
         'paymentInfo': {
             'name': request['paymentInfo']['name'],
             'cardType': request['paymentInfo']['cardType'],
```

Comment:
We are not sending the modified card num back with profile info? only when asked for payment info?

Reply:
Thanks.

Good catch.

Reply:
Fixed